docs/release_notes/v0.7.0.md

```diff
@@ -0,0 +1,239 @@
+  
+# Dapr 0.7.0
+ 
+We're happy to announce the release of Dapr 0.7.0! 
+
+This release adds new components, and focuses on improvements to tracing and service invocation.
+We also added a performance test for service invocation, fixed bugs, improved stability and added features to our long haul tests infrastructure.
```

Comment:
With the v0.7 release we focused on improvements to tracing and service invocation. This release also expands Dapr functionality with addition components. Continuing with the team of improved stability, we also added a performance test for service invocation, fixed bugs, and added features to our long haul tests infrastructure.

While we focused in this release on improvements to tracing and service invocation, we did also extend Dapr functionality with addition of new components , and 
We also added a performance test for service invocation, fixed bugs, improved stability and added features to our long haul tests infrastructure.

Adds extends Dapr functionality or introduces ... using

Reply:
What does "Continuing with the team of improved stability?"

Reply:
The following sentence in the original had the bit about `improved stability`. As Dapr is nearing v1.0 release I think we should start communicating A) the growing focus on stability B) impact of the customer validation. I'm coming late to this so feel free to ignore it